src/meta/src/barrier/command.rs

```diff
@@ -994,6 +994,21 @@ impl CommandContext {
                         init_split_assignment.clone(),
                     )
                     .await?;
+
+                // Apply the split changes in source manager.
+                self.barrier_manager_context
+                    .source_manager
+                    .drop_source_fragments(std::slice::from_ref(old_table_fragments))
+                    .await;
+                let source_fragments = new_table_fragments.stream_source_fragments();
+                self.barrier_manager_context
+                    .source_manager
+                    .apply_source_change(
+                        Some(source_fragments),
+                        Some(init_split_assignment.clone()),
```

Comment:
I am curious about what `init_split_assignment` stands for. Is it the latest assignment from prev table?